webapp-django/crashstats/api/jinja2/api/documentation.html

```diff
@@ -105,17 +105,17 @@ <h2><a href="#{{ endpoint.name }}">{{ endpoint.name }}</a></h2>
             {% endfor %}
           </tbody>
         </table>
-      {% else %}
-        <p>
-          <em>Takes no parameters</em>
-        </p>
       {% endif %}
 
       <!-- for starting a test drive -->
       <div class="run-test">
-        <button type="submit">Run Test Drive!</button>
-        <img src="{{ static('img/ajax-loader16x16.gif') }}" alt="Loading..." class="loading-ajax">
-        <button type="button" class="close">&times; Close</button>
+        {% if endpoint.test_drive %}
+          <button type="submit">Run Test Drive!</button>
+          <img src="{{ static('img/ajax-loader16x16.gif') }}" alt="Loading..." class="loading-ajax">
+          <button type="button" class="close">&times; Close</button>
+        {% else %}
+          <em>Test drive not supported.</em>
+        {%endif %}
```

Comment:
API endpoints with complicated POST payloads don't work with the test drive stuff, so I added indication that it wasn't supported.

It'd be interesting to get it working at some point, but not today.